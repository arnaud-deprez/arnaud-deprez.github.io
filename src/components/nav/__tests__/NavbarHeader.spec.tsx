import React from 'react'
import { render } from '@testing-library/react'
import { NavbarHeader } from '..'

describe('<NavbarHeader/>', () => {
  it('renders correctly', () => {
    const { asFragment } = render(
      <NavbarHeader
        author={{
          name: 'Arnaud Deprez',
          jobTitle: 'Technical Architect'
        }}
      />
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
