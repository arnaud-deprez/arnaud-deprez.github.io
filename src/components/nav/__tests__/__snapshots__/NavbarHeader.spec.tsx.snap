// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarHeader/> renders correctly 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-expand-lg navbar-dark bg-primary"
    id="navbar-header"
  >
    <div
      class="container-fluid"
    >
      <a
        class="d-lg-none navbar-brand"
        href="/"
      >
        Arnaud Deprez
      </a>
      <button
        aria-controls="navbar-header-menu"
        aria-label="Toggle navigation"
        class="btn btn-primary navbar-toggler collapsed"
        type="button"
      />
      <div
        class="justify-content-end navbar-collapse collapse"
        id="navbar-header-menu"
      >
        <ul
          class="navbar-nav"
          role="navigation"
        >
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/"
            >
               About
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/blog/"
            >
               Blog
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="/contact/"
            >
               Contact
            </a>
          </li>
        </ul>
        <hr
          class="d-lg-none"
        />
        <ul
          class="social-icons d-lg-none flex-row justify-content-center navbar-nav"
          role=""
        >
          <li
            class="nav-item"
          >
            <a
              aria-label="Rss"
              class="nav-link"
              href="/rss.xml"
              rel="noopener noreferrer"
              target="_blank"
            />
          </li>
        </ul>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
